src/objective.jl

```diff
@@ -160,3 +160,26 @@ function objective_function(model::Model,
                    MOI.ObjectiveFunction{MOIFunType}())::MOIFunType
     return jump_function(model, func)
 end
+
+"""
+    set_objective_coefficient(model::Model, term, coeff)
+
+Updates the objective function to change the coefficient of the given term.
+If the objective does not have the right type, an error will be thrown:
+changing a variable coefficient for a constant objective, changing the
+coefficient of a quadratic term where there is only a linear objective.
+"""
+function set_objective_coefficient end
+
+function set_objective_coefficient(model::Model, variable::VariableRef, coeff)
```

Comment:
There should be a specific docstring here instead of the generic one above:

```Julia
"""
    set_objective_coefficient(model::Model, variable::VariableRef, coefficient::Real)

Set the linear objective coefficient associated with `Variable` to `coefficient`.

Note: this function will throw an error if a nonlinear objective is set.
"""